src/main/resources/logback.xml

```diff
@@ -65,7 +65,7 @@
       </providers>
     </encoder>
     <filter class="ch.qos.logback.classic.filter.ThresholdFilter">
-      <level>DEBUG</level>
+      <level>INFO</level>
```

Comment:
Why was this changed?  This'll make it so that case only logs info lines right?  If so, we want it to log at debug level so we can have more and filter out what we don't need

Reply:
It is on DEBUG level which makes the log size really big on GCP for devs. We should only make it debug when it is required. It was wrong so changed it to info